packages/sakuli-legacy/src/context/sahi/action/common-action/common-actions.spec.ts

```diff
@@ -65,8 +94,12 @@ describe("common-actions", () => {
                 </ul>
             `)
         );
+
+        // WHEN
         await api._highlight(queryByLocator(By.css("#second")));
-        expect(scrollIntoViewModule.scrollIntoViewIfNeeded).toHaveBeenCalled();
+
+        // THEN
+        expect(scrollIntoViewSpy).toHaveBeenCalledTimes(1);
```

Comment:
One could verify the parameters passed to the scrollIntoView call. This would ensure that the browser scrolls to the requested element.